.gitignore

```diff
@@ -0,0 +1,19 @@
+# Ignore vi swapfiles
+*.swp
+.*.swp
+
+# Ignore Python cache directories
+__pycache__/
+
+# Ignore project coverage
+.coverage
+
+# Ignore tox working directory
+.tox/
```

Comment:
```suggestion
# Ignore project coverage
/.coverage

# Ignore tox working directory
/.tox/
```

because we only want to ignore these directories at the root of the repo